docs/basic/pgtypes.rst

```diff
@@ -273,7 +273,7 @@ database using:
 Because |hstore| is distributed as a contrib module, its oid is not well
 known, so it is necessary to use `!TypeInfo`\.\
 `~psycopg.types.TypeInfo.fetch()` to query the database and get its oid. The
-resulting object you can use passed to
+resulting object you can pass to
```

Comment:
How about:
```suggestion
resulting object can be passed
```

Reply:
+1 on Daniele's suggestion; I think that's cleaner than both the other variants.

Reply:
👍